src/frame/js/actions/http.js

```diff
@@ -0,0 +1,97 @@
+import urljoin from 'urljoin';
+
+/**
+ * Stringifies query parameters and append them to the url
+ * @param  {string} url  - an url
+ * @param  {object} data - an object containing the query parameters
+ * @return {string}      - the final url
+ */
+export function stringifyGETParams(url, data) {
```

Comment:
yeah we have those, I just forgot to port them over from core https://github.com/smooch/smooch-core-js/blob/master/test/specs/utils/http.spec.js